provision-virtualization-tools.sh

```diff
@@ -59,17 +62,17 @@ rm $terraform_libvirt_provider_filename
 
 # install Packer.
 apt-get install -y unzip
-packer_version=1.5.5
+packer_version=1.6.6
```

Comment:
was there a reason for not upgrading to 1.8.0?

Reply:
You rigth, I followed the instructions from some post on orden to resolve the error i've got from packer. They said that version Will work. I ll chance that